src/context/NotificationContext.tsx

```diff
@@ -2,77 +2,76 @@ import React, { useState, useCallback } from 'react';
 import Notification, { INotificationProps } from '../Alerts/atom/Notification';
```

Comment:
I did an auto-format from the vscode tool, there is actually no "real change in this file"